cmd/k8s-operator/operator.go

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Tailscale Inc & AUTHORS
 // SPDX-License-Identifier: BSD-3-Clause
 
+//go:build !plan9
```

Comment:
This feels burdensome if it grows, I assume this is to make ./... functional?

Really my question becomes a bit different: we have a community member who's been diligently working on making illumos work, and I wonder if we'd accept the same for that?

Reply:
Yes, this is for `./...` (which CI does).

I'm fine with adding `illumos` here. But that might not need it, being more Unix-y. (actually Unix?)

Reply:
Yeah, it may not raise the same thing. I need to follow up with that person sometime and see how their patches are. I know they've sent some and periodically the mention wanting to send more, but have some questions.